### ISSUE TITLE
How can I use Multiple y-axis to display dataset

### ISSUE THREAD

**Issue by griffinwbrown** [airbnb/visx#1318]
I need to set my right Y-axis to scale to a dollar amount, to stack on top of my left Y-axis values which are volume. Is this possible currently in visx? I feel like I need to map the axis oriented to the right to my third dataset but I don't see any documentation on how to do this. [
![multiline](https://user-images.githubusercontent.com/82471750/129794621-6750462c-234c-4de7-87f0-4b4041d715b5.PNG)
](url)

**Comment by griffinwbrown** [comment]
this is the code for my chart: 
```
<XYChart height={300} xScale={{ type: "band" }} yScale={{ type: "linear" }}>
      <AnimatedAxis orientation="bottom" />
      <AnimatedAxis orientation="left" axisClassName="1"/>
      <AnimatedAxis orientation="right" />
      <AnimatedGrid columns={false} numTicks={4} />
      <AnimatedLineSeries dataKey="Line 1" axisClassName="1" data={data1} {...accessors} />
      <AnimatedLineSeries dataKey="Line 2" axisClassName="1" data={data2} {...accessors} />
      <AnimatedLineSeries dataKey="Line 3"  data={data3} {...accessors} />
      <Tooltip
        snapTooltipToDatumX
        snapTooltipToDatumY
        showVerticalCrosshair
        showSeriesGlyphs
        renderTooltip={({ tooltipData, colorScale }) => (
          <div>
            <div style={{ color: colorScale(tooltipData.nearestDatum.key) }}>
              {tooltipData.nearestDatum.key}
            </div>
            {accessors.xAccessor(tooltipData.nearestDatum.datum)}
            {", "}
            {accessors.yAccessor(tooltipData.nearestDatum.datum)}
          </div>
        )}
      />
    </XYChart>
```

**Comment by williaster** [comment]
Hello! This duplicates https://github.com/airbnb/visx/issues/1189 

**tl;dr** dual y-axis charts are not recommended because they are difficult to interpret and can be misleading. If you choose this type of chart you'll need to use the lower-level `visx` packages to create it yourself with `@visx/axis` + `@visx/scale`
